docs/getting-started/csm-stack/csm-sandbox.ipynb

```diff
@@ -0,0 +1,257 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CSM Sandbox\n",
+    "\n",
+    "## 1. Prerequisites\n",
+    "\n",
+    "### Install Knoten\n",
+    "This tutorial requires Knoten version 0.4 or higher.\n",
+    "\n",
```

Comment:
Change the ISD to be a full CTX ISD, then add an explanation that this is a CTX, line scanner image.